<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisClustering</class>
 <widget class="QWidget" name="analysisClustering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clustering spots</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QtCharts::QChartView" name="plot"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxNormalization">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Different normalization methods for the counts</string>
       </property>
       <property name="statusTip">
        <string>Normalization methods</string>
       </property>
       <property name="whatsThis">
        <string>Different normalization methods for the counts</string>
       </property>
       <property name="title">
        <string>Normalization</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="normalization_raw">
          <property name="text">
           <string>RAW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="normalization_tpm">
          <property name="text">
           <string>TPM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="normalization_rel">
          <property name="text">
           <string>REL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="normalization_deseq">
          <property name="text">
           <string>DESeq2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="normalization_scran">
          <property name="text">
           <string>SCRAN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dimenstionality Reduction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Method</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="tSNE">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Dimenstionality by t-SNE (non-linear)</string>
          </property>
          <property name="statusTip">
           <string>Dimenstionality by t-SNE (non-linear)</string>
          </property>
          <property name="text">
           <string>t-SNE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="PCA">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Dimenstionality by PCA (linear)</string>
          </property>
          <property name="statusTip">
           <string>Dimenstionality by PCA (linear)</string>
          </property>
          <property name="text">
           <string>PCA</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Initial components (t-SNE)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="initial_pc">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The number if initial components to use in t-SNE</string>
         </property>
         <property name="statusTip">
          <string>The number if initial components to use in t-SNE</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Perplexity (t-SNE)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="perplexity">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The t-SNE perplexity value</string>
         </property>
         <property name="statusTip">
          <string>The t-SNE perplexity value</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Dimensions</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="dimenstions_2">
          <property name="cursor">
           <cursorShape>BlankCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Reduce to 2 main pricincipal components</string>
          </property>
          <property name="statusTip">
           <string>Reduce to 2 main pricincipal components</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dimensions_3">
          <property name="cursor">
           <cursorShape>SplitHCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Reduce to 3 main pricincipal components</string>
          </property>
          <property name="statusTip">
           <string>Reduce to 3 main pricincipal components</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clustering</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Method</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="kmeans">
          <property name="toolTip">
           <string>KMeans clustering (centroids)</string>
          </property>
          <property name="statusTip">
           <string>KMeans clustering (centroids)</string>
          </property>
          <property name="text">
           <string>KMeans</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hiearchical">
          <property name="toolTip">
           <string>Hierarchical clustering (Ward linkage)</string>
          </property>
          <property name="statusTip">
           <string>Hierarchical clustering (Ward linkage)</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Hierarchical</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Number of clusters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="clusters">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>The number of different clusters to make</string>
         </property>
         <property name="statusTip">
          <string>The number of different clusters to make</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="runClustering">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
